src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -61,7 +61,8 @@ protected override bool IsParentAnAttribute(SyntaxNode node)
                 TryClassifyFromIdentifier(name, symbolInfo, result) ||
                 TryClassifyValueIdentifier(name, symbolInfo, result) ||
                 TryClassifyNameOfIdentifier(name, symbolInfo, result) ||
-                TryClassifyAsyncIdentifier(name, symbolInfo, result);
+                TryClassifyAsyncIdentifier(name, symbolInfo, result) ||
+                TryClassifyPartialIdentifier(name, symbolInfo, result);
```

Comment:
Is the intent to classify any contextual keyword in this code path when it's not bound to a symbol? It might be a good idea then to have a single method that checks for all contextual keywords instead of keeping this list increasing with time.